rns_shell/compositor/layers/Layer.h

```diff
@@ -109,6 +110,7 @@ class Layer {
     void removeFromParent();
 
     virtual void paintSelf(PaintContext& context);
+    virtual void paintChildren(PaintContext& context);
```

Comment:
make the newly added functions as inline

Reply:
As discusssed, since ScrollLayer is also using these functions - making it inline will not have acess to the definition. So as discussed will keep it as non-line itself.